tracing/tracing.go

```diff
@@ -34,6 +34,14 @@ type StartConfig struct {
 
 type SpanOpt func(config *StartConfig)
 
+// WithSpanAttribute appends attributes to a new created span.
+func WithSpanAttribute(attributes ...attribute.KeyValue) SpanOpt {
```

Comment:
1. Let's rename this from `WithSpanAttribute` to `WithAttribute`. This is going to be used with `tracing.StartSpan`, so it's already clear that the attributes are going to be applied to span and no need to repeat that.
2. Let's have it accept `k string, v interface{}` instead of `attributes ...attribute.KeyValue`. This way we'll have it more ergonomic (less nested function calls):


Before:
```go
	ctx, span := tracing.StartSpan(ctx, "task.Kill",
 		tracing.WithSpanAttribute(
 			tracing.Attribute("task.id", t.ID()),
 			tracing.Attribute("task.pid", int(t.Pid()))))
```

After:
```go
	ctx, span := tracing.StartSpan(ctx, "task.Kill",
 		tracing.WithAttribute("task.id", t.ID()),
 		tracing.WithAttribute("task.pid", int(t.Pid())),
        )
```

WDYT?

Reply:
Sounds good. I have made the change acordingly